pkg/sdk/logging/model/syslogng/output/http.go

```diff
@@ -65,8 +66,11 @@ type HTTPOutput struct {
 	// The value of the USER-AGENT header in the messages sent to the server.
 	UserAgent string `json:"user-agent,omitempty"`
 	// Description: Specifies the number of worker threads (at least 1) that syslog-ng OSE uses to send messages to the server. Increasing the number of worker threads can drastically improve the performance of the destination.
-	Workers     int    `json:"workers,omitempty"`
-	PersistName string `json:"persist_name,omitempty"`
+	Workers        int                `json:"workers,omitempty"`
+	PersistName    string             `json:"persist_name,omitempty"`
+	LogFifoSize    int                `json:"log-fifo-size,omitempty"`
```

Comment:
```suggestion
	LogFIFOSize    int                `json:"log-fifo-size,omitempty"`
```
follow naming guidelines https://github.com/golang/go/wiki/CodeReviewComments#initialisms